### PR TITLE
Create ui-c8y-1020-7-3-fixing-issue-with-resizing-of-data-point-widget-6708-graft-release.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-59814](https://cumulocity.atlassian.net/browse/MTM-59814)
Original PR: [6734](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6734)

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1020-7-3-fixing-issue-with-resizing-of-data-point-widget-6708-graft-release.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: fixing issue with resizing of data point widget (#6708) [GRAFT][release/cd] (#6734)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-7-3-fixing-issue-with-resizing-of-data-point-widget-6708-graft-release.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Fixed resizing of data point widget
```

File: content/change-logs/application-enablement/ui-c8y-1020-7-3-fixing-issue-with-resizing-of-data-point-widget-6708-graft-release.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-59814
+version: 1020.7.3
+---
+fixing issue with resizing of data point widget (#6708) [GRAFT][release/cd] (#6734)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-7-3-fixing-issue-with-resizing-of-data-point-widget-6708-graft-release.md:17]: The Claude AI suggested to use the following body:
```suggestion
In the past, resizing the data point widget in dashboards sometimes led to unexpected behavior and layout issues. This change fixes the resizing of data point widgets to ensure they maintain their correct size and position within the dashboard layout. With this improvement, users can now reliably resize data point widgets without encountering layout problems, enabling them to create and customize their dashboards more effectively.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Review by janhommes — APPROVED** [review]

File: content/change-logs/application-enablement/ui-c8y-1020-7-3-fixing-issue-with-resizing-of-data-point-widget-6708-graft-release.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-59814
+version: 1020.7.3
+---
+In the past, resizing the data point widget in dashboards sometimes led to unexpected behavior and layout issues. This change fixes the resizing of data point widgets to ensure they maintain their correct size and position within the dashboard layout. With this improvement, users can now reliably resize data point widgets without encountering layout problems, enabling them to create and customize their dashboards more effectively.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-7-3-fixing-issue-with-resizing-of-data-point-widget-6708-graft-release.md:17]:
```suggestion
In the past, resizing the "Data point" widget in dashboards sometimes led to unexpected behavior and layout issues. This change fixes the resizing of "Data point" widgets to ensure they maintain their correct size and position within the dashboard layout. 
```

File: content/change-logs/application-enablement/ui-c8y-1020-7-3-fixing-issue-with-resizing-of-data-point-widget-6708-graft-release.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Fixed resizing of data point widget
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-7-3-fixing-issue-with-resizing-of-data-point-widget-6708-graft-release.md:3]:
```suggestion
title: Fixed resizing of "Data point" widget
```

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**